demo_numpy.py

```diff
@@ -0,0 +1,94 @@
+# Copyright 2021 D-Wave Systems Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+import dimod
+from dwave.system import LeapHybridSampler
+
+import demo
+
+def build_bqm(potential_new_cs_nodes, num_poi, pois, num_cs, charging_stations, num_new_cs):
+    """ Build bqm that models our problem scenario using NumPy. """
```

Comment:
I realize that it's not in the original demo either, but looking at this now, it would be nice to expand this docstring to include parameter definitions + types. Would make the demo easier to understand imo. (Would be good to add it to demo.py as well)